i18n/es.json

```diff
@@ -4,6 +4,7 @@
 			"AlvaroMolina"
 		]
 	},
+	"action-managewiki": "administrar la configuración de wiki",
```

Comment:
You should add "una" to the Spanish Translation so that the description is consistent.